hack/fetch-all-latest-and-push.sh

```diff
@@ -37,22 +37,23 @@ repos=(
     apiextensions-apiserver
     apimachinery
     apiserver
-    client-go
     cli-runtime
+    client-go
     cloud-provider
     cluster-bootstrap
     code-generator
     component-base
     component-helpers
```

Comment:
had to create the following repos in `kubernetes-nightly`:
- component-helpers
- controller-manager